packages/dcos-integration-test/extra/test_tweeter_demo.py

```diff
@@ -0,0 +1,102 @@
+import logging
+import os
+
+import pytest
+import requests
+import retrying
+
+import test_helpers
+
+from test_packaging import (
+    skipif_insufficient_resources
+)
+
+log = logging.getLogger(__name__)
+
+DEMO_RESOURCE_REQUIREMENTS = {
+    'number_of_nodes': 5,
+    'node': {
+        'disk': 0,
+        'mem': 0,
+        'cpus': 1
+    }
+}
+
+
+def _get_tweeter_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/tweeter.json')
+    return response.json()
+
+
+def _get_post_tweets_app_definition():
+    response = requests.get('https://raw.githubusercontent.com/mesosphere/tweeter/master/post-tweets.json')
+    return response.json()
+
+
+def _get_tweet_count(address):
+    """Count the number of tweets on the tweeter app's webpage
+
+    Args:
+        address: str URL
+    """
+    response = requests.get(address)
+    return response.text.count('tweet-content')
+
+
+@retrying.retry(
+    wait_fixed=10000,
+    stop_max_delay=60000)
+def _wait_for_tweet_count(url):
+    # This is 25 tweets because of how paging in tweeter is defined
+    assert _get_tweet_count(url) == 25, "Did not find expected 25 tweets"
+
+
+@pytest.mark.skipif(
+    test_helpers.expanded_config.get('security') == 'strict',
```

Comment:
I'm not sure why this just checks for strict. It should skip this if it's in any EE environment.